scalding-repl/src/main/scala/com/twitter/scalding/ReplImplicits.scala

```diff
@@ -58,12 +67,62 @@ trait BaseReplState {
     useHdfsMode_()
     customConfig -= mr1Key
     customConfig -= mr2Key
+    modeBanner
   }
 
   def useHdfsLocalMode() {
     useHdfsMode_()
     customConfig += mr1Key -> mrLocal
     customConfig += mr2Key -> mrLocal
+    modeBanner
+  }
+
+  def modeBanner: Unit = {
+    val scaldingMode = mode match {
```

Comment:
can we do

`val (modeString, homeDir) =`

so it is more clear what is in each part of the tuple?